sdx-connector/src/main/java/com/sequenceiq/cloudbreak/saas/sdx/SaasSdxService.java

```diff
@@ -96,4 +97,9 @@ public StatusCheckResult getAvailabilityStatusCheckResult(SDXSvcCommonProto.Inst
     public Optional<Entitlement> getEntitlement() {
         return Optional.of(CDP_SAAS_SDX_INTEGRATION);
     }
+
+    @Override
+    public SdxClusterResponse getSdxClusterByEnvironmentCrn(String environmentCrn) {
```

Comment:
paas related API response is SaasSdxService, definitely not good, can you create a followup jira @RokolyA94 to fix this?